components/presence/presence.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <div
+    v-if="presence"
```

Comment:
I made `presence` take a default, we probably don't need `v-if` in this case, right?

Reply:
no, since it will always be true.